runtime/cudaq/builder/kernel_builder.h

```diff
@@ -444,24 +453,27 @@ class kernel_builder : public details::kernel_builder_base {
     QuakeValue v(*opBuilder, param);                                           \
     details::NAME(*opBuilder, v, empty, qubit);                                \
   }                                                                            \
-  template <NumericType ConstantParameter>                                     \
-  void NAME(ConstantParameter &&constant, QuakeValue qubit) {                  \
-    NAME(constant, qubit);                                                     \
-  }                                                                            \
-  template <typename mod,                                                      \
+  template <typename mod, QuakeValueOrNumericType paramT,                      \
```

Comment:
Would you mind capitalizing this type - ParamT

Reply:
Definitely. Fixed.